nix/nixos/cardano-node-options.nix

```diff
@@ -109,7 +110,7 @@ in {
 
       stateDir = mkOption {
         type = types.str;
-        default = "/var/lib/cardano-node";
+        default = "/var/lib/cardano-node-%i";
```

Comment:
lets not pollute lib. Maybe optionally if multiple instances are running, do `/var/lib/cardano-node/%i`

Reply:
Will do!

Reply:
Fixed.